server/app/api/v1/charity/controller.js

```diff
@@ -0,0 +1,108 @@
+import {
+  NotFoundError, AlreadyInUseError, ValidationError
+} from 'common-errors';
+import merge from 'lodash/merge';
+import Charity from './model';
+
+import { VALID_OBJECT_ID } from '../../../util/validations';
```

Comment:
The path for this should be: `../../../constants` -- makes it easier to identify constants and change them in one spot.